docs/packages/pkg/corrade.rst

```diff
@@ -0,0 +1,42 @@
+.. spelling::
+
+    corrade
+
+.. index::
+  single: graphics ; corrade
+
+.. _pkg.corrade:
+
+corrade
+===
```

Comment:
We need `=======` instead of `===` here.

Reply:
Fixed in 432b6f4505ad83062a765c472ceef2ecfeb3ced0